app/src/main/res/values-de/strings.xml

```diff
@@ -934,7 +934,6 @@ Bei Gruppenunterhaltungen kann der Gruppen-Admin diese Einstellung überschreibe
     <string name="settings_show_typing_indicator_description">Wenn diese Option aktiviert ist, sehen Sie, wenn andere Personen eine Nachricht schreiben, und andere sehen, wenn Sie tippen. Gilt für alle Unterhaltungen auf diesem Gerät.</string>
     <string name="settings_app_lock_title">Mit Kennwort sperren</string>
     <string name="settings_send_anonymous_usage_data_title">Anonyme Nutzungsdaten senden</string>
-    <string name="settings_send_anonymous_usage_data_description">Anhand der Nutzungsdaten kann Wire verstehen, wie die App genutzt wird und wie sie verbessert werden kann. Die Daten sind anonym und umfassen nicht den Inhalt Ihrer Kommunikation (wie Nachrichten, Dateien, Standorte oder Anrufe).</string>
```

Comment:
why did we remove it completely?

Reply:
Because if the base, strings.xml has changes, that requires a new translation, removing the old one